SluggoTests/TagTests.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  TagTests.swift
+//  SluggoTests
+//
+//  Created by Samuel Schmidt on 5/11/21.
+//
+
+import XCTest
+@testable import Sluggo
+
+class TagTests: XCTestCase {
+
+    let workingSingleJson = """
+    {
+        "id": 1,
+        "team_id": 15,
+        "object_uuid": "a40cb545-3ada-4656-a02d-2fdaba81fc21",
+        "title": "word around office, the",
```

Comment:
would this not fail because tags are not supposed to have spaces?

Reply:
Old test, forgot to update the title. Sanity checking is done on the backend so it will serialize correctly on the front end.